tdrs-backend/tdpservice/parsers/schema_defs/header.py

```diff
@@ -58,74 +62,23 @@
         Field(
             item="1",
             name="state_fips",
+            friendly_name="state fips",
             type="string",
             startIndex=12,
             endIndex=14,
             required=True,
             validators=[
-                validators.oneOf(
-                    (
-                        "01",
-                        "02",
-                        "04",
-                        "05",
-                        "06",
-                        "08",
-                        "09",
-                        "10",
-                        "11",
-                        "12",
-                        "13",
-                        "15",
-                        "16",
-                        "17",
-                        "18",
-                        "19",
-                        "20",
-                        "21",
-                        "22",
-                        "23",
-                        "24",
-                        "25",
-                        "26",
-                        "27",
-                        "28",
-                        "29",
-                        "30",
-                        "31",
-                        "32",
-                        "33",
-                        "34",
-                        "35",
-                        "36",
-                        "37",
-                        "38",
-                        "39",
-                        "40",
-                        "41",
-                        "42",
-                        "44",
-                        "45",
-                        "46",
-                        "47",
-                        "48",
-                        "49",
-                        "50",
-                        "51",
-                        "53",
-                        "54",
-                        "55",
-                        "56",
-                        "66",
-                        "72",
-                        "78",
-                    )
-                )
+                validators.oneOf(("01", "02", "04", "05", "06", "08", "09", "10", "11", "12", "13", "15",
```

Comment:
this might create a conflict with @elipe17 's [PR](https://github.com/raft-tech/TANF-app/pull/2742/files) or the other way around, depending on which gets merged first. We can cross that bridge when we get to it.

Reply:
I don't think his branch is touching header file, so when we merge this, his branch can pull develop and should be ok